packages/flutter/lib/src/cupertino/icons.dart

```diff
@@ -21,11 +21,11 @@ import 'package:flutter/widgets.dart';
 ///   cupertino_icons: ^1.0.0
```

Comment:
To reviewer(s), everything after line 1077 is automatically generated and does not require review